### DISCUSSION TITLE
The window blinks a lot before the plot is shown

### DISCUSSION THREAD

**Discussion by yurivict** [alandefreitas/matplotplusplus#223] [Ideas]
With the gnuplot backend the plot in the window blinks many times before the plot stabilizes.

**Comment by alandefreitas** [comment]
It's not possible to get high-frequency updates with gnuplot, because it blinks when the pipes are updating the plot. The only option is not using draw/show more than needed ("quiet" figures), not updating as much, or coming up with another backend.

**Comment by vlsireddy** [comment]
Hi ,
     i use subplot(3,1,0), subplot(3,1,1), subplot(3,1,2)
A. every 200 millisecond, each of these is called in succession [back to back]. still i see flickering issue.
B. I see same flickering issue with Tiledlayout as well.

Q1. plz let me know how to overcome this issue ?
Q2. Also where exactly is this issue originating [i see people say use draw_idle in matplot, what is the equivalent in matplot++] ?

**Reply by alandefreitas** [reply]
This cannot be overcome with gnuplot. You can improve it a little by not drawing the intermediary plots. Use the figure in quiet mode (see docs) and only call `draw` when the complete figure is assembled.

The possibility of new backends besides gnuplot has been discussed extensively. It seems like this will need to wait until matplot++ v2.

**Reply by diverger** [reply]
@alandefreitas May you tell me what's the alternative backend which can bring us less flickering?

**Reply by alandefreitas** [reply]
I'm sorry. There's no good solution for now.

**Comment by vlsireddy** [comment]
Hi,
    i used quiet mode and was able to solve this issue.
Thanks
